pkg/controller/clusterpool/clusterpool_controller.go

```diff
@@ -26,18 +27,20 @@ import (
 
 	apihelpers "github.com/openshift/hive/pkg/apis/helpers"
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+	hiveintv1alpha1 "github.com/openshift/hive/pkg/apis/hiveinternal/v1alpha1"
 	"github.com/openshift/hive/pkg/clusterresource"
 	"github.com/openshift/hive/pkg/constants"
 	hivemetrics "github.com/openshift/hive/pkg/controller/metrics"
 	controllerutils "github.com/openshift/hive/pkg/controller/utils"
 )
 
 const (
-	ControllerName             = hivev1.ClusterpoolControllerName
-	finalizer                  = "hive.openshift.io/clusters"
-	imageSetDependent          = "cluster image set"
-	pullSecretDependent        = "pull secret"
-	credentialsSecretDependent = "credentials secret"
+	ControllerName                   = hivev1.ClusterpoolControllerName
+	finalizer                        = "hive.openshift.io/clusters"
+	imageSetDependent                = "cluster image set"
+	pullSecretDependent              = "pull secret"
+	credentialsSecretDependent       = "credentials secret"
+	hibernateAfterSyncSetsNotApplied = time.Minute * 10
```

Comment:
Some documentation to indicate what this variable is doing will be really helpful

Reply:
This has moved to the hibernation controller but I added a comment for it.